recipes/rank_filter/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "rank_filter" %}
+{% set version = "0.4.11" %}
+{% set checksum = "246f5c73ebd03314d9b2cff3ede74330" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ checksum }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - boost >=1.56
+    - python
+    - setuptools >=18.0
+    - numpy x.x
```

Comment:
Interesting, for some reason I was thinking they would need to be on the same line. Thanks. Have updated accordingly.